# best_function_ever
pass your exception to google_problem(), it will google and open first stackoverflow page itself 
